test/test_map_train.py

```diff
@@ -64,7 +65,7 @@ def test_run_train_models(self):
           [LogisticRegression, classification_error, None, (), {'lam':50}]]
 
         errors = run_train_models(2, model_library, X=self.X, y=self.y)
-        for k in errors.keys():
+        for k in list(errors.keys()):
```

Comment:
You can drop this call to `list`, but the ones in the asserts above are necessary.